standalone/soroban-rpc/etc/stellar-captive-core.cfg

```diff
@@ -7,6 +7,7 @@ PUBLIC_HTTP_PORT=false
 PEER_PORT=11726
 DATABASE="__DATABASE__"
 ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING=__ARTIFICIALLY_ACCELERATE_TIME_FOR_TESTING__
+ENABLE_SOROBAN_DIAGNOSTIC_EVENTS=__ENABLE_SOROBAN_DIAGNOSTIC_EVENTS__
```

Comment:
can this be enabled on the captive core config for futurenet also?

Reply:
It is enabled for futurenet as well, yes.